baseapp_payments/rest_framework/customers/serializers.py

```
@@ -0,0 +1,78 @@
+            "entity_type",
+            "entity_id",
+        )
+
+    def validate(self, data):
+        if "user_id" in data:
+            try:
+                user = get_user_model().objects.get(id=data["user_id"])
+                data["user"] = user
+                return data
+            except get_user_model().DoesNotExist:
+                raise serializers.ValidationError("User does not exist.")
```

Ruff (0.15.7)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)